bridetalk_project/src/components/pages/MasterLayout.jsx

```diff
@@ -3,24 +3,35 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import { HomePage } from "./homepage/HomePage";
 import { LoginPage } from "./login/LogInPage";
 import { CreateUserPage } from "./create-user/CreateUserPage";
+import { ResetPage } from "./reset-password/ResetPage";
+import { Footer, Navbar } from "../molecule";
+
+import "./masterlayout.css"
 
 export const MasterLayout = () => {
```

Comment:
Interesting approach to extract the routes outside the App. The name confuses me a little bit tho. 
What is **MasterLayout**? It makes me think of a styled box wrapping your application, while it is a component that provides the routes. 
Maybe a Route-related name?